desktop/renderer/style.css

```diff
@@ -139,3 +139,21 @@ body {
 	display: none;
 	width: 0;
 }
+
+.fade-anim-enter {
+  opacity: 0.01;
```

Comment:
I just copy/pasted the react animation example, but looking into it a bit more:

https://twitter.com/aliukani/status/725752918075396098

Not a lot of information, but I think it might force gpu rendering ?

We shouldn't need it, I'll change